File: week04/week04-6.py
class Solution:
    def pivotInteger(self, n: int) -> int:
        total=0
        for i in range(1,n+1):
            total+=i
        left=0
        for i in range(1,n+1):
            left+=i
            if left + left-i == total:return i
        return -1

File: week04/week04-6.cpp
int pivotInteger(int n) {
    int total=0;
    for(int i=1; i<=n;i++){
        total+=i;
    }
    int left=0;
    for(int i=1; i<=n; i++){
        left+=i;
        if(left+left-i==total) return i;
    }
    return -1;
}